### PR TITLE
Workaround for an infinite recursion bug

### PR DESCRIPTION
**Pull request by Chris-Turnbull**
Hi there

In version 0.6 you could pass a jQuery element into the steps array. This still works fine in version 1... except the new _cloneObject() function goes into an infinite recursion loop if you do this.

I've written a workaround which seems to work fine and fix the problem for us.

### CODE REVIEW

File: intro.js [hunk 1]
```diff
@@ -262,7 +262,11 @@
       }
       var temp = {};
       for (var key in object) {
-          temp[key] = _cloneObject(object[key]);
+          if (object[key] instanceof jQuery) {
```
Comment (afshinm) [inline intro.js]: Here you need to check if the jQuery is defined. I think this line can throw an error.

Comment (Chris-Turnbull) [inline intro.js]: Oh, my apologies for that, we always include jQuery on our pages so I didn't notice.

Comment (afshinm) [inline intro.js]: Cool!

**Comment by afshinm** [comment]
merged.

**Pull request closed without merging**